lib/services/local/users.go

```diff
@@ -63,30 +63,41 @@ var GlobalSessionDataMaxEntries = 5000 // arbitrary
 // user accounts as well
 type IdentityService struct {
 	backend.Backend
-	log        logrus.FieldLogger
-	bcryptCost int
+	log              logrus.FieldLogger
+	bcryptCost       int
+	notificationsSvc *NotificationsService
 }
 
 // NewIdentityService returns a new instance of IdentityService object
-func NewIdentityService(backend backend.Backend) *IdentityService {
-	return &IdentityService{
-		Backend:    backend,
-		log:        logrus.WithField(teleport.ComponentKey, "identity"),
-		bcryptCost: bcrypt.DefaultCost,
+func NewIdentityService(backend backend.Backend) (*IdentityService, error) {
```

Comment:
This is going to break e/ builds on master and backport release branches. Could we do this in a way that we don't break the build?